evcxr/src/eval_context.rs

```diff
@@ -102,6 +102,108 @@ pub(crate) struct Config {
     subprocess_path: PathBuf,
 }
 
+#[derive(Default)]
+pub(crate) struct InitConfig {
+    tmpdir: Option<PathBuf>,
+    pub(crate) init: Option<String>,
+    pub(crate) prelude: Option<String>,
+}
+
+impl InitConfig {
+    pub(crate) fn check_if_exists(path: &Path) -> bool {
+        if !path.is_dir() {
+            return false
+        }
+        path
+            .read_dir()
```

Comment:
Rather than reading the directory to see if it contains the file, what about just doing `path.join("init.evcxr").exists()`?

Reply:
It seems more reasonable and logical, thank you very much.